spec/temporal.html

```diff
@@ -1,51 +1,77 @@
-<!doctype html>
-<meta charset="utf8">
-
-<emu-clause id="sec-temporal-object">
+<emu-clause id="sec-temporal-objects">
   <h1>The Temporal Object</h1>
-  <p>
-    The Temporal object is the <dfn>%Temporal%</dfn> intrinsic object and the initial value of the *"Temporal"* property of the global object.
-    The Temporal object is a single ordinary object.
-  </p>
-  <p>
-    The value of the [[Prototype]] internal slot of the Temporal object is the intrinsic object %ObjectPrototype%.
-  </p>
-  <p>
-    The Temporal object is not a function object.
-    It does not have a [[Construct]] internal method; it is not possible to use the Temporal object as a constructor with the *new* operator.
-    The Temporal object does not have a [[Call]] internal method; it is not possible to invoke the Temporal object as a function.
-  </p>
+  <p>The Temporal object:</p>
+  <ul>
+    <li>is the intrinsic object <dfn>%Temporal%</dfn>.</li>
+    <li>is the initial value of the *"Temporal"* property of the global object.</li>
+    <li>is a single ordinary object.</li>
```

Comment:
Pre-existing, so feel free to punt to a follow-up if you like: what does "single" mean here? I searched ECMA-262 for the phrase "single ordinary object" and didn't find it anywhere else.

Also, I noticed that ECMA-262 has nifty links to the definition of 'ordinary object', I managed to get the same by adding this to the biblio file:
```json
    {
      "type": "term",
      "term": "ordinary object",
      "refId": "ordinary-object",
      "id": "ordinary-object"
    }
```
But again this is pre-existing so feel free to ignore it for now.

Reply:
Yeah, I didn't find "single" anywhere either, but I decided not to change it. I'll make these improvements in a PR now.